.vscode/tasks.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build",
+            "type": "shell",
+            "command": "make build",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            },
+            "problemMatcher": [
+                "$go"
+            ]
+        },
+        {
+            "label": "Webdriver Single Chrome Test",
+            "type": "shell",
+            "command": "go test -v -timeout=15m -tags=large ./webdriver -run ${input:testName} -args -firefox_path=/usr/bin/firefox -geckodriver_path=$GECKODRIVER_PATH -chrome_path=/usr/bin/google-chrome -chromedriver_path=/usr/bin/chromedriver -frame_buffer=false -staging=false -browser=chrome",
+            "group": {
+                "kind": "test",
+                "isDefault": true
+            },
+            "problemMatcher": [
+                "$go"
+            ]
+        },
+        {
+            "label": "Webdriver Chrome Tests",
```

Comment:
On a new environment, chrome and chromedriver were not installed. I had to run `make chrome && make chromedriver`. You could make a new task like the following: 

```
        {
            "label": "Install WebDriver Deps",
            "type": "shell",
            "command": "make chrome && make chromedriver",
        },
```

Then in this task and the single chrome test task, you can use the dependsOn section like so:

```json
        {
            "label": "Webdriver Chrome Tests",
            "type": "shell",
            "command": "go test -v -timeout=15m -tags=large ./webdriver -args -chrome_path=/usr/bin/google-chrome -chromedriver_path=/usr/bin/chromedriver -frame_buffer=false -staging=false -browser=chrome",
            "group": {
                "kind": "test",
                "isDefault": true
            },
            "dependsOn": [
                "Install WebDriver Deps"
            ],
            "problemMatcher": [
                "$go"
            ]
        }
```

This will make sure the user has everything installed beforehand.

Reply:
Done.